uk_bin_collection/uk_bin_collection/councils/ConwyCountyBorough.py

```
@@ -11,9 +11,41 @@ class CouncilClass(AbstractGetBinDataClass):
+            )
+        }
+        # www.conwy.gov.uk intermittently returns connection resets, read
+        # timeouts, or SSL handshake failures depending on load. Do two
+        # short attempts so a flaky request still fits the production budget.
+        import time as _time
+        response = None
+        last_err = None
+        for attempt in range(2):
+            try:
+                response = requests.get(
+                    uri, headers=headers, timeout=15, verify=False,
```

Ruff (0.15.9)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)